.gitignore

```diff
@@ -20,3 +20,4 @@ build/*
 #IDE and code editor(s) setting
 .idea/
 .vscode
+.obsidian
```

Comment:
@ar-mokhtari 

why ignored obsidian configurations?

Reply:
I think it is personal.
when I edit document(s), it's not for others, is it?
@Ja7ad

Reply:
@ar-mokhtari 
This is required workspace for write book, it's easier to write a book.
We required use obsidian app for write book in https://github.com/GoFarsi/book/blob/main/COUNTRIBUTING.md#%D9%86%D8%AD%D9%88%D9%87-%D9%85%D8%B4%D8%A7%D8%B1%DA%A9%D8%AA

Reply:
Sure, but we don't talk about "obsidian", we are talking about "obsidian config".
Every time I try to make pull request, change of obsidian "config" want to be in pull request. I think it can be different for every one.
However if you think it must be in  pull request every time any one send, I will add it ...

Reply:
> Sure, but we don't talk about "obsidian", we are talking about "obsidian config". Every time I try to make pull request, change of obsidian "config" want to be in pull request. I think it can be different for every one. However if you think it must be in pull request every time any one send, I will add it ...

If want use obsidian personal configuration, better update COUNTRIBUTING rules for obsidian step.